config/initializers/devise.rb

```diff
@@ -0,0 +1,278 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '4e69f36dde0bcbc616e46ddb3938978a7f65cc7ef1c4fc47bfed313e7215d12f2288388155bb6a204120ac1cb627ced842f8917bfc7bc73d13dea2773be8ba31'
```

Comment:
That's a commented out fake key. It's all good. Thanks for paying attention though as that would  be super bad to have that for real in the repo.